tests/unit_tests/test_tethys_apps/test_base/test_workspace.py

```diff
@@ -42,72 +42,72 @@ class TethysAppChild(tethys_app_base.TethysAppBase):
 
 class TestUrlMap(unittest.TestCase):
     def setUp(self):
-        self.root = os.path.abspath(os.path.dirname(__file__))
-        self.test_root = os.path.join(self.root, "test_workspace")
-        self.test_root_a = os.path.join(self.test_root, "test_workspace_a")
-        self.test_root2 = os.path.join(self.root, "test_workspace2")
+        self.root = Path(__file__).parent.absolute()
+        self.test_root = self.root / "test_workspace"
+        self.test_root_a = self.test_root / "test_workspace_a"
+        self.test_root2 = self.root / "test_workspace2"
         self.app = tethys_app_base.TethysAppBase()
         self.user = UserFactory()
 
     def tearDown(self):
-        if os.path.isdir(self.test_root):
+        if self.test_root.is_dir():
             shutil.rmtree(self.test_root)
-        if os.path.isdir(self.test_root2):
+        if self.test_root2.is_dir():
             shutil.rmtree(self.test_root2)
 
     def test_TethysWorkspace(self):
         # Test Create new workspace folder test_workspace
-        result = base_workspace.TethysWorkspace(path=self.test_root)
+        result = base_workspace.TethysWorkspace(path=str(self.test_root))
```

Comment:
Doesn't `TethysWorkspace` accept a `Path` object?

Reply:
So `TethysWorkspace` will accept a `Path`. With your changes it just converts it to a `Path` first think anyway. However, it's not worth changing since this works fine and the workspaces API is going to be dropped in 5.0.